en/mkdocs.yml

```diff
@@ -113,9 +113,13 @@ nav:
     - EMR Connectivity:
       - EMR Systems Overview: emr-connectivity/guides/emr-systems-overview.md
       - Connecting with Epic EMR systems: emr-connectivity/guides/emr-connector.md
-    - Securing a Health APIs:
+    - Securing Health APIs:
       - SMART on FHIR: secure-health-apis/guides/smart-on-fhir.md
-      - Retrive Patient ID: secure-health-apis/guides/getting-patientid.md
+      - Configure SMART App Launch: secure-health-apis/guides/getting-patientid.md
+    - Advance Topics:
```

Comment:
```suggestion
    - Advanced Topics:
```

Reply:
Fixed in https://github.com/wso2/open-healthcare-docs/pull/45/commits/bc3c197716619abfbc8849f455e744b9f0f18df5